utils/utils.jsx

```diff
@@ -968,11 +971,12 @@ export function changeCss(className, classValue) {
     const styleSheet = styleEl.sheet;
     const rules = styleSheet.cssRules || styleSheet.rules;
     const style = classValue.substr(0, classValue.indexOf(':'));
-    const value = classValue.substr(classValue.indexOf(':') + 1);
+    const value = classValue.substr(classValue.indexOf(':') + 1).replace(/!important[;]/g, '');
```

Comment:
Are these changes just to support `!important`? I'm not too familiar with the API being used here